### PR TITLE
same sized dropdown boxes

### PR DESCRIPTION
**Pull request by DanielZavala11**

### CODE REVIEW

**Comment by DanielZavala11** [comment]
issue #18 conform with feedback

File: src/navbar/navbar.css [hunk 1]
```diff
@@ -148,3 +151,31 @@
...
+	padding-left: 22px;
+	padding-right: 24px;
+	margin: 0;
+}
+
+#resources-navbar-link {
+	padding-left: 22px;
+	padding-right: 22px;
+	margin-left: 0;
+}
```
Comment (KHVBui) [inline src/navbar/navbar.css:181]: Add a return here at the end of the file.

**Review by KHVBui — CHANGES_REQUESTED** [review]
While you're working on the navbar, could you also fix the console error on line 59 in "NavigationBar.js"?
```
Warning: Received `false` for a non-boolean attribute `className`.
If you used to conditionally omit it with className={condition && value}, pass className={condition ? value : undefined} instead.
```

Otherwise it all looks good!

**Comment by KHVBui** [comment]
Also don't forget to format your project with prettier so that it can successfully build.

**Review by KHVBui — APPROVED** [review]
Good to go!

**Pull request merged**